### ISSUE TITLE
Enable axe tooling

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-react#603]
As a developer, I'd like to have a way to check my work for accessibility violations as I'm developing so that I can resolve issues in real time as they arise. [react-axe](https://github.com/dequelabs/react-axe) looks like a good option and provides a more intimate developer experience as checks are ran on state change as opposed to manual click to run.

Let's integrate with react-axe so that developers can find and address these issues at development time without having to run an entire audit or spot test individual pages manually.
